nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java

```diff
@@ -64,18 +66,27 @@ void shouldContainDistinctDescriptions() {
         assertThat(duplicatesMessage, distinctValues, is(equalTo(ontologyValues)));
     }
 
-    @Test
-    void shouldContainEveryVisibleClassOfModel() {
+    @ParameterizedTest
+    @MethodSource("provideModelClassesArguments")
```

Comment:
Yep, this is a more constructive way of doing this :) call the method modelClassProvider.

Reply:
fixed

Reply:
And yes, it is easier to see what class/property is the problem this way.